src/stream_management/mod_stream_management.erl

```diff
@@ -615,14 +621,21 @@ get_all_stanzas_to_forward(StateData, SMID) ->
     LServer = mongoose_c2s:get_lserver(StateData),
     FromServer = jid:make_noprep(<<>>, LServer, <<>>),
     ToForward = [ begin
-                      TS = mongoose_acc:timestamp(Acc),
-                      Packet = mongoose_acc:element(Acc),
-                      StanzaName = mongoose_acc:stanza_name(Acc),
-                      StanzaType = mongoose_acc:stanza_type(Acc),
-                      maybe_add_timestamp(Packet, StanzaName, StanzaType, TS, FromServer)
+                      AccWithTS = maybe_add_timestamp(Acc, FromServer),
+                      {_From, _To, El} = mongoose_acc:packet(AccWithTS),
+                      El
```

Comment:
mongoose_acc:element(Acc) will give you the element more cleanly than with packet.

Reply:
Ahh, thanks, let me change that.

Reply:
I amended the previous commit